src/central_control/full_system.launch

```diff
@@ -15,8 +15,8 @@
 <!-- We're not using this node for now because the sign and car recognition are the same node for now -->
 <!--   <node name="car_detect" pkg="sign_car_recognition" type="car_detect.py" /> -->
   <node name="sign_detect" pkg="sign_car_recognition" type="sign_detect.py" />
-  <!-- name="object_detect" pkg="lidar_object_recognition" type="lidar_object_detect.py" /> -->
-  <node name="new_object_detect" pkg="lidar_object_recognition_2" type="new_lidar_object_detect.py"/>
+  <node name="object_detect" pkg="lidar_object_recognition" type="lidar_object_detect.py" />
```

Comment:
Do we need to revert to the old object detection?

Reply:
Me and robell did go through this and since the new lidar is not fine-tuned yet, it would run smoother when using the old one. We can use the new one once I finished fine tune it

Reply:
yup exactly what brian said. When running the paths with the new lidar detection, the vehicle would deviate a lot from path even though the vehicle wasnt colliding with objects, reverting to the old lidar for now.

Reply:
Aha, makes sense. Interested to see how improvements to lidar can make this even better!